src/Contract/Operation/Zipable.php

```diff
@@ -22,9 +22,9 @@ interface Zipable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#zip
      *
-     * @param iterable<mixed> ...$iterables
+     * @param iterable<mixed, mixed> ...$iterables
      *
-     * @return Collection<TKey,T>
+     * @return Collection<list<mixed|TKey>, list<mixed|T>>
```

Comment:
```suggestion
     * @return Collection<list<mixed>, list<mixed>>
```

`mixed` already covers TKey and T, so this could be simplified as such.